app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -30,15 +30,18 @@
  */
 public class EtorDomainRegistration implements DomainConnector {
 
+    public static final String DEMOGRAPHICS_API_ENDPOINT = "/v1/etor/demographics";
+    public static final String ORDERS_API_ENDPOINT = "/v1/etor/orders";
```

Comment:
You can make these "package private" which will allow them to be used in tests but not outside the tests.

```suggestion
    static final String DEMOGRAPHICS_API_ENDPOINT = "/v1/etor/demographics";
    static final String ORDERS_API_ENDPOINT = "/v1/etor/orders";
```